### DISCUSSION TITLE
[Question] can this able to detect chat ID spoofing?

### DISCUSSION THREAD

**Discussion by CoolLKKPS** [Charlese2/HostFixes#8] [Q&A] [answered]
so my friend encounter cheater using my ID to chat with him, does this mod can prevent that?
* not only just the client who installed this mod, but also effect hole lobby if host installed this mod
* this should not have additional config options, this patch will apply as long as installed this mod

**Comment by Charlese2** [comment, marked as answer]
It does check that the ID used to chat matches the player that sent the RPC.
If the host has the mod installed it will prevent clients from sending messages that look like it is from someone else.

https://github.com/Charlese2/HostFixes/blob/a653c3c4b639917eb1b55e9a500f0e11bb7d2480/HostFixes/Plugin.cs#L470-L477
